src/docfx/build/toc/parser/TableOfContentsParser.cs

```diff
@@ -21,10 +21,14 @@ internal static class TableOfContentsParser
 
         public delegate (string content, Document file) ResolveContent(Document relativeTo, SourceInfo<string> href, bool isInclusion);
 
+        public delegate List<string> ResolveMoniker(Document doc);
+
         public static (List<Error> errors, TableOfContentsModel model)
-            Load(Context context, Document file, ResolveContent resolveContent, ResolveHref resolveHref, ResolveXref resolveXref)
+            Load(Context context, Document file, ResolveContent resolveContent, ResolveHref resolveHref, ResolveXref resolveXref, ResolveMoniker resolveMoniker)
```

Comment:
Consider use `context.DependencyResolver` and `context.MonikerProvider` to save these delegates and parameters. Could be in another PR.
```suggestion
            Load(Context context, Document file)
```

Reply:
TODo

---
In reply to: [293207838](https://github.com/dotnet/docfx/pull/4702#discussion_r293207838) [](ancestors = 293207838)